### DISCUSSION TITLE
Is there a way to obtain the reply-to ID from Github Issue E-mail Notification before another person

### DISCUSSION THREAD

**Discussion by carlosparadis** [community/community#23319] [New to GitHub]
<p>I have the following situation:</p>
<ol>
<li>Github Issue is created on website</li>
<li>(a few weeks go by with co-workers/supervisors unaware of it as the notification go to their “Forum” tab or they are e-mail only)</li>
<li>Eventually one kind soul responds on the issue, generating an e-mail notification to me, the issue creator. In there, a reply-to address is provided containing &lt;organization_on_github&gt;/&lt;project_on_github&gt;&lt;reply+&lt;a_crazy_long_id_I_am_after&gt;@reply.github.com&gt;</li>
</ol>
<p>At that point, I can grab the <code>\&lt;organization_on_github\&gt;/\&lt;project_on_github\&gt;\&lt;reply+\&lt;a_crazy_long_id_I_am_after\&gt;@reply.github.com\&gt;</code> e-mail, and start e-mailing the team with it Cc. As people reply to all, I can get all their messages stored in the issue tracker from the e-mail thread.<br>
I am trying to skip steps 2 and 3 and get that ID somehow after the issue is created. That way, I can send them an e-mail after creating the issue with it Cc. Is there a way to do that?<br>
Note my interest is not on creating Github issues from my e-mail, just being able to get that issue generated ID by the e-mail notification, without having to hope someone responds to the e-mail so I get the notification.<br>
The only information I could find online was an article from 2011:<br>
<a href="https://github.blog/2011-03-10-reply-to-comments-from-email/" rel="nofollow noopener">https://github.blog/2011-03-10-reply-to-comments-from-email/</a><br>
when Github apparently implemented said functionality. I find it hard to believe 9 years later I can’t skip steps 2 and 3.<br>
Worst comes to end, I am willing to even switch to another service like GitLab or Bitbucket, if you happen to know that is available for a free account. I believe Slack allows to do something of the sort, where each channel has an e-mail id where you can Cc, and messages go to that channel. But I really could use this for an issue tracker and a VCS.<br>
Thank you!</p>

**Comment by guntrip** [comment, marked as answer]
<p>Hello there Carlos!</p>
<p>Hmm, could you try enabling “Include your own updates” in your “Email notification preferences”? You’ll find that on the following page:</p>
<p><a href="https://github.com/settings/notifications" rel="nofollow noopener">https://github.com/settings/notifications</a></p>
<p>I’ve just tested this and confirmed that you recieve an email upon creating an issue (or any other content that triggers a notification) and that email contains the reply-to address you’re looking for. Any emails sent to that address will be added as comments from your GitHub account.</p>
<p>Otherwise, I’m afraid it isn’t possible to obtain the address and token associated with an issue. Instead, I’ve logged this as a feature request in our internal feature request list. Though I can’t guarantee anything or share a timeline for this, I can tell you that it’s been shared with the appropriate teams for consideration.</p>
<p>I hope that’s a helpful workaround!</p>

**Reply by mirabilos** [reply]
Could you please implement this then? This would be **super**-useful for eMail-based collaboration workflows.
